blazor/datepicker/islamic-calendar.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: Islamic Calendar in Blazor DatePicker Component | Syncfusion
+description: Checkout and learn here all about Islamic Calendar in the Syncfusion Blazor DatePicker component and much more.
+platform: Blazor
+control: DatePicker
+documentation: ug
+---
+
+# Islamic Calendar in Blazor DatePicker Component
+
+In addition to the Gregorian calendar, the DatePicker control supports displaying the Islamic calendar (Hijri calendar). Islamic calendar or Hijri calendar is a lunar calendar consisting of 12 months in a year of 354 or 355 days.Users can either select a date from the Islamic calendar or manually enter a date. To know more about Islamic calendar [wikipedia](https://en.wikipedia.org/wiki/Islamic_calendar).
+
+Also, it consists of all Gregorian calendar functionalities as like min and max date, week number, start day of the week, multi selection, enable RTL, start and depth view, localization, highlight and customize the specific dates.By default, calendar mode is in Gregorian. You can enable the Islamic mode by setting the calendarMode as Islamic.
```

Comment:
inform about the ConvertToHijri and ConvertToGregorian methods to parse from and to hijirr